src/convertToObject.js

```diff
@@ -7,6 +7,13 @@
  */
 function convertToObject(sourceString) {
   // write your code here
+  return sourceString.split(';').reduce((prev, next) => {
+    const [key, value] = next.split(':').map((item) => item.trim());
+
+    prev[key] = value;
+
+    return prev;
```

Comment:
The current implementation of `reduce` will fail if there are empty or incomplete declarations in the string. It's important to add a check to ensure that both `key` and `value` are defined before adding them to the accumulator object.